cargo-dist/src/backend/installer/homebrew.rs

```diff
@@ -0,0 +1,43 @@
+//! Code for generating installer.sh
+
+use axoasset::LocalAsset;
+use serde::Serialize;
+
+use super::InstallerInfo;
+use crate::{
+    backend::templates::{Templates, TEMPLATE_INSTALLER_RB},
+    errors::DistResult,
+    installer::ExecutableZipFragment,
+};
+
+/// Info about a Homebrew formula
+#[derive(Debug, Clone, Serialize)]
+pub struct HomebrewInstallerInfo {
+    /// The application's name
+    pub name: String,
+    /// Formula class name
+    pub formula_class: String,
+    /// The application's license, in SPDX format
+    pub license: Option<String>,
+    /// The URL to the application's homepage
+    pub homepage: Option<String>,
+    /// A brief description of the application
+    pub desc: Option<String>,
+    /// AMD64 artifact
+    pub x86_64: Option<ExecutableZipFragment>,
```

Comment:
I figured that since Homebrew does need to specifically query the x86_64 and ARM binaries, it made sense to promote them to first-class properties of the installer info struct.